webview/platforms/qt.py

```diff
@@ -235,19 +235,18 @@ def interceptRequest(self, info):
     class NavigationHandler(QWebPage):
         def __init__(self, page):
             super().__init__(page.profile())
-            self.parent = page.parent
+            self._parent = page.parent
 
         def acceptNavigationRequest(self, url, type, is_main_frame):
             if settings['OPEN_EXTERNAL_LINKS_IN_BROWSER']:
                 webbrowser.open(url.toString(), 2, True)
                 return False
             else:
-                self.parent.load_url(url.toString())
+                self._parent.load_url(url.toString())
```

Comment:
I haven't observed this breaking anything, but it seemed susceptible to the same issue so I eagerly fixed. Happy to revert.